dashboard/app/controllers/policy_compliance_controller.rb

```diff
@@ -0,0 +1,86 @@
+class PolicyComplianceController < ApplicationController
+  before_action :authenticate_user!, except: [:child_account_consent]
+
+  # GET /policy_compliance/child_account_consent
+  # URL where parents or guardians give consent for their child to have an
+  # account on Code.org
+  def child_account_consent
+    @permission_granted = false
+    token = request.params[:token]
+    return render status: :bad_request if token.blank?
+    #Get parent permission request
+    permission_request = ParentalPermissionRequest.find_by(uuid: token)
```

Comment:
`params.require(:token)` will raise `ActionController::ParameterMissing` if `token` is blank and return `bad_request` HTTP status.
```suggestion
    #Get parent permission request
    permission_request = ParentalPermissionRequest.find_by(uuid: params.require[:token])
```

Reply:
This change is in Dayne's part of the code and seems to have been addressed already. Sorry for the confusing diffs... I've had to base this off of his PR so it shows some of his diffs, too, when they don't align with staging. I've rebased and that should clear it up.